CHANGELOG.md

```diff
@@ -3,6 +3,43 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.6.0] - 2024-05-06
+
+### Added
+- Simple RAG Example build a basic Retrieval-Augmented Generation (RAG) system.
+- Easily switch from [API Catalog](https://build.nvidia.com/explore/discover) models to locally deployed models using NVIDIA's enterprise solution (NIM) for enhanced production use.
+- New APIs
+  - `/health` - Provides a health check for the chain server.
+  - `/delete` - Allows deletion of ingested documents.
+  - `/list` - Lists all ingested documents.
+- Evaluator support to quantify performance of RAG pipeline
```

Comment:
`A new evaluation application to quantify accuracy of different provided RAG examples.`
Point to the directory here.

Reply:
Replaced with Containerized [evaluation application](./tools/evaluation/) for RAG pipeline accuracy measurement as suggested by Sumit in other comment